yarn-project/aztec-rpc/src/synchroniser/synchroniser.ts

```diff
@@ -138,6 +138,11 @@ export class Synchroniser {
 
   protected async workNoteProcessorCatchUp(limit = 1, retryInterval = 1000): Promise<void> {
     const noteProcessor = this.noteProcessorsToCatchUp[0];
+    if (this.synchedToBlock === 0) {
```

Comment:
Does it make sense to make this more general i.e. 

```
      if (noteProcessor.status.syncedToBlock === this.synchedToBlock) {
        // Note processor already synched, nothing to do
        this.noteProcessorsToCatchUp.shift();
        this.noteProcessors.push(noteProcessor);
        return;
      }
```

Reply:
Addressed in [cd3b19d](https://github.com/AztecProtocol/aztec-packages/pull/1427/commits/cd3b19dc50e1c65f55359f62e8668e8e9d388484). Thanks